octobot_trading/personal_data/orders/order.py

```diff
@@ -217,9 +227,20 @@ async def initialize_impl(self, **kwargs):
         Initialize order status update tasks
         """
         await orders_states.create_order_state(self, **kwargs)
+        self.is_initialized = True
```

Comment:
Shouldn't it be set by `Initializable.initialize()`?

Reply:
I had to set it here as well because when orders are filled instantly (ex market orders), it's happening within this `initialize` call, therefore an order can be filled before `is_initialized` is the to True. If we use `is_initialized` in `is_open`, then we need to set it before initializing the order state.

Also, I'm wondering if using `is_initialized` is the best idea to check if an order is open, maybe we should use something else (the original need for this is to know that an order has been initialized/created to know if we should cancel it when referenced as linked order. Issue being that wrapped order might not have been initialized/created at this point). 
Refactoring linked orders in a way I described on discord would remove the need for this by the way.

Reply:
Thanks for the explanation